tests/system/testdata/deployments/state.txtar

```diff
@@ -37,14 +37,16 @@ project:
   triggers:
     - name: http_get_short
       connection: my_connection
-      call: program.star:on_http_get_short
+      event_type: get
```

Comment:
is there a way to define unite event, e.g. for both GET and POST?

Reply:
Not in `event_type` - you can either use:
1. Two triggers pointing to the same `call` function
2. Single trigger with a `filter` that accepts both event types

Reply:
so, if I want trigger to be triggered on GET I need to use `event_type`, and if I want it to be triggered on both Get and POST I need to use filter?

Reply:
Yes. You can also say that you can always use a filter, but if you want to avoid "complicated" CEL syntax you can use event_type.